core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -77,8 +78,9 @@ object ConfigCommand extends Logging {
 
   val BrokerDefaultEntityName = ""
   val BrokerLoggerConfigType = "broker-loggers"
-  val BrokerSupportedConfigTypes = ConfigType.all :+ BrokerLoggerConfigType :+ ConfigType.ClientMetrics
-  val ZkSupportedConfigTypes = Seq(ConfigType.User, ConfigType.Broker)
+  @nowarn("cat=deprecation")
+  val BrokerSupportedConfigTypes = JavaConverters.asScalaIteratorConverter(ConfigType.ALL.iterator()).asScala.toSeq :+ BrokerLoggerConfigType :+ ConfigType.CLIENT_METRICS
```

Comment:
Why are we using an iterator here? Can we use `val BrokerSupportedConfigTypes: Seq[String] = ConfigType.ALL.asScala :+ BrokerLoggerConfigType :+ ConfigType.CLIENT_METRICS`?

Reply:
Great catch. Thanks. Rewrote like you suggested.